src/pipelines.js

```diff
@@ -550,6 +550,42 @@ export class TextGenerationPipeline extends Pipeline {
  * trained on NLI (natural language inference) tasks. Equivalent of `text-classification`
  * pipelines, but these models don't require a hardcoded number of potential classes, they
  * can be chosen at runtime. It usually means it's slower but it is **much** more flexible.
+ * 
+ * **Example:** Zero shot classification with `Xenova/bart-large-mnli` (one text).
```

Comment:
Have you tried the other zero-shot classification models on the hub? https://huggingface.co/models?pipeline_tag=zero-shot-classification&library=transformers.js&sort=trending Does bart-large-mnli work best for you? If possible, it might be good to use a smaller model for the example:

- [bart-large-mnli](https://huggingface.co/Xenova/bart-large-mnli/tree/main/onnx) (411MB unquantized; 1.63GB quantized)
- [mobilebert-uncased-mnli ](https://huggingface.co/Xenova/mobilebert-uncased-mnli/tree/main/onnx) (27MB quantized; 99MB unquantized)
- [distilbert-base-uncased-mnli](https://huggingface.co/Xenova/distilbert-base-uncased-mnli/tree/main/onnx) (68MB quantized; 268MB unquantized)

Reply:
Yes good idea to have a smaller model for the example 👍